README.md

```diff
@@ -784,6 +784,41 @@ let app =
     ]
 ```
 
+#### Gotchas
+
+If your route is not returning static html, then you should wrap your function with a warbler.
+
+```fsharp
+// unit -> string
+let model() =
+    System.DateTime.Now.ToString()
+
+// HtmlNode
+let staticView =
+    p [] (model() |> rawText)
+
+// unit -> HtmlNode
+let dynamicView() =
+    p [] (model() |> rawText)
+
+let webApp = 
+    choose [
+        GET >=>
+            choose [
+                route "/static"      >=> (staticView |> renderHtml)
+                route "/dynamic"     >=> warbler (fun _ -> dynamicView() |> renderHtml)
+            ]
+    ]
+```
+
+Functions in F# are eagerly evaluated and in the case of razor/htmlEngine a warbler will help to evaluate the function every time.
+
+```fsharp
+// ('a -> 'a -> 'b) -> 'a -> 'b
+let warbler f a =
+    f a a
+```
+
```

Comment:
Yes that is true... 
I was thinking about the big README and so far I actually thought its not so bad. As long as the index is always updated it has actually quite some advantages, because people can easily find stuff in a singe document by simple ctrl+f search, easier edit the document with find/replace and so on. I see it a bit like a big word document, which is essentially also just one file.

I will think about a separate wiki though and perhaps move things out from the README...